apis/v1alpha2/shared_types.go

```diff
@@ -506,14 +325,14 @@ type AnnotationValue string
 // The `NamedAddress` value has been deprecated in favor of implementation
 // specific domain-prefixed strings.
 //
-// All other values, including domain-prefixed values have Custom support, which
-// are used in implementation-specific behaviors. Support for additional
+// All other values, including domain-prefixed values have Implementation-specific support,
+// which are used in implementation-specific behaviors. Support for additional
 // predefined CamelCase identifiers may be added in future releases.
 //
 // +kubebuilder:validation:MinLength=1
 // +kubebuilder:validation:MaxLength=253
 // +kubebuilder:validation:Pattern=`^Hostname|IPAddress|NamedAddress|[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*\/[A-Za-z0-9\/\-._~%!$&'()*+,;=:]+$`
-type AddressType string
+type AddressType = v1beta1.AddressType
 
 const (
```

Comment:
similar const copy issue.

Reply:
Good catch, I'm removing consts from v1a2 entirely in #1576